prime-dotnet-webapi-tests/Unit Tests/LookupService.Tests.cs

```diff
@@ -57,5 +58,48 @@ public async void TestGetLookups()
             Assert.NotEmpty(results.HealthAuthorities);
             Assert.NotEmpty(results.Facilities);
         }
+
+        [Fact]
+        public async void TestLookupsLicenseFilter()
+        {
+            var service = MockDependenciesFor<LookupService>(DefaultMapper());
+
+            var results = await service.GetLookupsAsync();
+            Assert.NotNull(results);
+
+            Assert.NotEmpty(results.Licenses);
+            Assert.Equal(new LicenseConfiguration().SeedData.Count(), results.Licenses.Count);
+
+
+            var future = new License { Code = 1000, Weight = 1, Name = "Future License" };
+            var futureDetail = new LicenseDetail { Id = 1000, LicenseCode = 1000, EffectiveDate = DateTime.UtcNow.AddDays(1), Prefix = "91", Manual = false, Validate = true, NamedInImReg = true, LicensedToProvideCare = true };
+
+            var past = new License { Code = 2000, Weight = 2, Name = "Past License" };
+            var pastDetail = new LicenseDetail { Id = 2000, LicenseCode = 2000, EffectiveDate = DateTime.UtcNow, Prefix = "91", Manual = false, Validate = true, NamedInImReg = true, LicensedToProvideCare = true };
+
+            // Load Test Data
+            TestDb.Add(future);
+            TestDb.Add(futureDetail);
+            TestDb.Add(past);
+            TestDb.Add(pastDetail);
+
+            TestDb.SaveChanges();
+
+            var newResults = await service.GetLookupsAsync();
+
+            Assert.NotNull(newResults);
+            Assert.NotEmpty(newResults.Licenses);
+            Assert.Equal(new LicenseConfiguration().SeedData.Count() + 1, newResults.Licenses.Count);
+            Assert.Empty(newResults.Licenses.Where(l => l.Code == future.Code));
+            Assert.NotEmpty(newResults.Licenses.Where(l => l.Code == past.Code));
+
+            // Clean Up Test Data
```

Comment:
You should not have to do this at the end of your test like this. I'm pretty sure the records wont be there in the next test as is. If records _are_ being left behind, we need a global solution, not having to remove the records at the end of each test like this.

Reply:
ahh true, i though i tried it and got an error but must have been something else.